service/src/main/kotlin/no/nav/su/se/bakover/service/søknad/lukk/LukkSøknadServiceImpl.kt

```diff
@@ -137,8 +138,10 @@ internal class LukkSøknadServiceImpl(
                     log.error("Kunne ikke lage brevutkast siden vi ikke fant personen for søknad ${request.søknadId}")
                     KunneIkkeLageBrevutkast.FantIkkePerson
                 }.flatMap { person ->
+                    val sak = sakService.hentSak(fnr = person.ident.fnr)
```

Comment:
Trenger kanskje ikke lenger hente søknaden dersom vi henter saken

Reply:
sliter litt med at jeg må hente søknaden for å finne fnr eller sakid som jeg trenger for å hente sak, men har endret nå til å kalle funksjonen hentSakidOgSaksnummer som kun henter dette